### PR TITLE
FIX: UltiSnips search only searches snippet value (not description) when using :Snippets command.

### PR DESCRIPTION
**Pull request by whatever555**
Before: (nothing is returned when searching by snippet description)   
image

  
Now (can search by both description and snippet):  
image
and  
image
  
  
Related issue: https://github.com/junegunn/fzf.vim/issues/1050

### CODE REVIEW

**Comment by whatever555** [comment]
This has been open for a while with no feedback. Should I close this PR?

**Comment by junegunn** [comment]
Sorry for the late reply. I don't use UltiSnips anymore so I haven't had a chance to test this. The patch looks good, but I would change it to something like `-n 1,..` so you can still perform anchored search e.g. `^foo`, `bar$`
What do you think?

**Comment by whatever555** [comment]
apologies for the late reply:  yep, I can change that now.

File: autoload/fzf/vim.vim [hunk 1]
```diff
@@ -964,7 +964,7 @@ function! fzf#vim#snippets(...)
   let colored = map(aligned, 's:yellow(v:val[0])."\t".v:val[1]')
   return s:fzf('snippets', {
   \ 'source':  colored,
-  \ 'options': '--ansi --tiebreak=index +m -n 1 -d "\t"',
+  \ 'options': '--ansi --tiebreak=index +m -n 1,2 -d "\t"',
```
Comment (whatever555) [inline autoload/fzf/vim.vim:967]: should that be ok ^

Comment (junegunn) [reply autoload/fzf/vim.vim:967]: No, I meant the literal `..` which means the whole range.

* `-n 1,..`: Let fzf search either on the first token only or on the whole string.

**Comment by junegunn** [comment]
Merged, thanks.

**Pull request merged**